Examples/Logs Sample/main.swift

```diff
@@ -18,7 +18,7 @@ func configure() {
       eventLoopGroup: MultiThreadedEventLoopGroup(numberOfThreads: 1))
 
       
-  OtlpLogExporter(channel: ClientConnection(configuration: configuration))
+  _ = OtlpLogExporter(channel: ClientConnection(configuration: configuration))
```

Comment:
I think this line should be deleted directly as the object is being recreated later in the place where it is used

Reply:
https://github.com/open-telemetry/opentelemetry-swift/pull/424/commits/92ce53a1447e0c519c87afcb2935c3b7cb68e4b2